Source/CesiumRuntime/Private/CesiumEncodedFeaturesMetadata.cpp

```diff
@@ -716,67 +672,49 @@ EncodedPropertyTexture encodePropertyTextureAnyThreadPart(
       if (pMappedUnrealImageIt) {
         encodedProperty.pTexture = pMappedUnrealImageIt->Pin();
       } else {
-        encodedProperty.pTexture = MakeShared<LoadedTextureResult>();
-        // TODO: upgrade to new texture creation path.
-        encodedProperty.pTexture->textureSource = LegacyTextureSource{};
-        propertyTexturePropertyMap.Emplace(pImage, encodedProperty.pTexture);
-        // This assumes that the texture's image only contains one byte per
-        // channel.
-        encodedProperty.pTexture->pTextureData = createTexturePlatformData(
-            pImage->width,
-            pImage->height,
-            EPixelFormat::PF_R8G8B8A8_UINT);
-
         const CesiumGltf::Sampler* pSampler = property.getSampler();
+
+        TextureAddress addressX;
         switch (pSampler->wrapS) {
         case CesiumGltf::Sampler::WrapS::REPEAT:
-          encodedProperty.pTexture->addressX = TextureAddress::TA_Wrap;
+          addressX = TextureAddress::TA_Wrap;
           break;
         case CesiumGltf::Sampler::WrapS::MIRRORED_REPEAT:
-          encodedProperty.pTexture->addressX = TextureAddress::TA_Mirror;
+          addressX = TextureAddress::TA_Mirror;
         case CesiumGltf::Sampler::WrapS::CLAMP_TO_EDGE:
         default:
-          encodedProperty.pTexture->addressX = TextureAddress::TA_Clamp;
+          addressX = TextureAddress::TA_Clamp;
         }
```

Comment:
Can we make this a helper function, since it comes up so often? Could put it in `CesiumTextureUtility`